doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,329 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
+    | Possible values: 'all', 'audit', 'auth', 'priv', 'ddl', 'dml', 'data_operations', 'compatibility',
+      'audit_enable', 'auth_ok', 'auth_fail', 'disconnect', 'user_create', 'user_drop', 'role_create', 'role_drop',
+      'user_disable', 'user_enable', 'user_grant_rights', 'role_grant_rights', 'role_revoke_rights', 'password_change',
+      'access_denied', 'eval', 'call', 'space_select', 'space_create', 'space_alter', 'space_drop', 'space_insert',
+      'space_replace', 'space_delete', 'custom'
+    | Default: 'nil'
+    | Environment variable: TT_AUDIT_LOG_FILTER
+
+..  _configuration_reference_audit_format:
+
+..  confval:: audit_log.format
+
+    Specify a format that is used for the audit log.
+
+    **Example**
+
+    If you set the option to ``plain``,
+
+    ..  code-block:: yaml
+
+        audit_log:
+            to: file
+            format: plain
+
+    the output in the file might look as follows:
+
+    ..  code-block:: text
+
+        2024-01-17T00:12:27.155+0300
+        4b5a2624-28e5-4b08-83c7-035a0c5a1db9
+        INFO remote:unix/:(socket)
+        session_type:console
+        module:tarantool
+        user:admin
+        type:space_create
+        tag:
+        description:Create space Bands
+
+    |
+    | Type: string
+    | Possible values: 'json', 'csv', 'plain'
+    | Default: 'json'
+    | Environment variable: TT_AUDIT_LOG_FORMAT
+
+..  _configuration_reference_audit_nonblock:
+
+..  confval:: audit_log.nonblock
+
+    Specify the logging behavior if the system is not ready to write.
+    If set to ``true``, Tarantool does not block during logging if the system is non-writable and writes a message instead.
+    Using this value may improve logging performance at the cost of losing some log messages.
+
+    ..  note::
+
+        The option only has an effect if the :ref:`audit_log.to <configuration_reference_audit_to>` is set to ``syslog``
+        or ``pipe``.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_AUDIT_LOG_NONBLOCK
+
+..  _configuration_reference_audit_pipe:
+
+..  confval:: audit_log.pipe
+
+    Specify a pipe for the audit log destination.
+    You can set the ``pipe`` type using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+    If log is a program, its pid is stored in the ``audit_log.logger_pid`` variable.
+    You need to send it a signal to rotate logs.
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log_pipe/config.yaml
+        :language: yaml
+        :start-at: audit_log:
+        :end-at: '| cronolog audit_tarantool.log'
+        :dedent:
+
+    This starts the `cronolog <https://linux.die.net/man/1/cronolog>`_ program when the server starts
+    and sends all ``audit_log`` messages to cronolog standard input (``stdin``).
+    If the ``audit_log`` string starts with '|',
+    the string is interpreted as a Unix `pipeline <https://en.wikipedia.org/wiki/Pipeline_%28Unix%29>`_.
```

Comment:
A pipeline connected to process'es stdin, I guess?

So, if there is no pipe at start, where the logs will be send?

(These my questions as a reader, I don't remember how it works.)